.circleci/config.yml

```diff
@@ -448,9 +448,15 @@ jobs:
           name: Install build dependencies
           command: |
             apt-get -qq update
-            apt-get -qy install clang-7 cmake libboost-regex-dev libboost-filesystem-dev libboost-test-dev libboost-system-dev libboost-program-options-dev libcvc4-dev libbz2-dev ninja-build zlib1g-dev libjsoncpp-dev=1.7.4-\*
+            apt-get -qy install wget clang-7 libboost-regex-dev libboost-filesystem-dev libboost-test-dev libboost-system-dev libboost-program-options-dev libcvc4-dev libbz2-dev ninja-build zlib1g-dev libjsoncpp-dev=1.7.4-\*
+            # evmone requires cmake version >= 3.13
```

Comment:
ubuntu disco should have 3.13.4-1 - did you try that?

Reply:
Ah nice, will update the image and see if it works.